.circleci/config.yml

```diff
@@ -270,8 +308,7 @@ jobs:
     steps:
       - checkout
       - npm-dependencies
-      - revenuecat/install-gem-mac-dependencies:
```

Comment:
So I am thinking we could leave this as is, and add the part that installs ruby right before